site/en/codelabs/openthread-border-router-nat64/index.lab.md

```diff
@@ -359,6 +359,70 @@ $ sudo ot-ctl nat64 mappings
 Done
 ```
 
+## Forward DNS queries to upstream DNS servers
+Duration: 03:00
+
+
+When NAT64 is enabled on the border router, OpenThread will try to forward the DNS queries for internet domains to upstream DNS servers.
+
+This function is supported by the internal DNSSD Server, so you need to ensure the DNSSD server is enabled.
+
+```console
+$ sudo ot-ctl srp server state
+running
+Done
+```
+
+If it is not "`running`", run `sudo ot-ctl srp server enable` to enable it.
```

Comment:
Just `running` (no " ") is fine

Also, the command to enable should be in a code block:

If it is not `running`, then enable it:

```console
$ sudo ot-ctl srp server enable
<don't know what the response is>
```

Just so users will know exactly what to expect on success.